CHANGELOG.md

```diff
@@ -15,7 +16,7 @@ recommendations of [keepachangelog.com](http://keepachangelog.com/).
 
 ### Fixed
 
-- None
+- The "item" association will no longer be required when belongs_to_required_by_default is enabled.
```

Comment:
Please follow the convention of this file and put in a link to the PR, so people can find out more about the change if they have questions.